ted_sws/notice_validator/entrypoints/cli/cmd_sparql_runner.py

```diff
@@ -46,6 +46,8 @@ def __init__(
         repository_path = Path(self.mappings_path)
         mapping_suite_repository = MappingSuiteRepositoryInFileSystem(repository_path=repository_path)
         self.mapping_suite = mapping_suite_repository.get(reference=self.mapping_suite_id)
+        self.log("K :: " + str(repository_path))
```

Comment:
what is this? 👯

Reply:
:D some development radiation. Thanks for observing